.github/workflows/release-js.yaml

```diff
@@ -0,0 +1,60 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+
+name: Release JS Library
+run-name: Publish Release ${{github.ref_name}} JS Library
+concurrency:
+  group: ${{github.workflow}}-${{github.ref}}
+  cancel-in-progress: false
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # ex. v1.0.0
+      - "v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+" # ex. v1.1.0-rc1
+      - "v0.0.1" # used for testing only
+      - "v0.0.1-rc[0-9]+" # used for testing only
+
+env:
+  NEW_RELEASE_TAG: ${{github.ref_name}}
+  TEST_RUN: ${{startsWith(github.ref_name, 'v0.0.1')}}
+
+jobs:
+  publish-npm:
+    name: Publish NPM Package
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [20.x, 22.x]
```

Comment:
Are we sure we want a matrix strategy here? Won't this overwrite itself with the same package version tag for both Node versions?